figures/README.md

```diff
@@ -0,0 +1,19 @@
+|`histology_color_by_sample.tsv`|Same histology grouping colors as noted above.|Lists hex codes by `Kids_First_Biospecimen_ID` instead of by histology group |For color-coding by `short_histology` when it's more convenient to assign colors based on `Kids_First_Biospecimen_ID`.|
+|`gradient_col_palette.tsv`| <br>gradient_0:![f7f7f7](https://placehold.it/150x40/f7f7f7/000000?text=f7f7f7) <br>gradient_1:![f7fcf5](https://placehold.it/150x40/f7fcf5/000000?text=f7fcf5) <br>gradient_2:![e5f5e0](https://placehold.it/150x40/e5f5e0/000000?text=e5f5e0) <br>gradient_3:![c7e9c0](https://placehold.it/150x40/c7e9c0/000000?text=c7e9c0) <br>gradient_4:![a1d99b](https://placehold.it/150x40/a1d99b/FFFFFF?text=a1d99b) <br>gradient_5:![74c476](https://placehold.it/150x40/74c476/FFFFFF?text=74c476) <br>gradient_6:![41ab5d](https://placehold.it/150x40/41ab5d/FFFFFF?text=41ab5d) <br>gradient_7:![238b45](https://placehold.it/150x40/238b45/FFFFFF?text=238b45) <br>gradient_8:![006d2c](https://placehold.it/150x40/006d2c/FFFFFF?text=006d2c) <br>gradient_9:![00441b](https://placehold.it/150x40/00441b/FFFFFF?text=00441b) <br>na_color:![f1f1f1](https://placehold.it/150x40/f1f1f1/000000?text=f1f1f1)|10 hex_codes where gradient_0 is for an absolute `0` but may need to be removed from the palette depending on the application|For numeric data being plotted e.g. tumor mutation burden|
+|`divergent_col_palette.tsv`|<br>divergent_low_5:![67001f](https://placehold.it/150x40/67001f/FFFFFF?text=67001f) <br>divergent_low_4:![b2182b](https://placehold.it/150x40/b2182b/FFFFFF?text=b2182b) <br>divergent_low_3:![d6604d](https://placehold.it/150x40/d6604d/FFFFFF?text=d6604d) <br>divergent_low_2:![f4a582](https://placehold.it/150x40/f4a582/FFFFFF?text=f4a582) <br>divergent_low_1:![fddbc7](https://placehold.it/150x40/fddbc7/000000?text=fddbc7) <br>divergent_neutral:![f7f7f7](https://placehold.it/150x40/f7f7f7/000000?text=f7f7f7) <br>divergent_high_1:![d1e5f0](https://placehold.it/150x40/d1e5f0/000000?text=d1e5f0) <br>divergent_high_2:![92c5de](https://placehold.it/150x40/92c5de/FFFFFF?text=92c5de) <br>divergent_high_3:![4393c3](https://placehold.it/150x40/4393c3/FFFFFF?text=4393c3) <br>divergent_high_4:![2166ac](https://placehold.it/150x40/2166ac/FFFFFF?text=2166ac) <br>divergent_high_5:![053061](https://placehold.it/150x40/053061/FFFFFF?text=053061) <br>na_color:![f1f1f1](https://placehold.it/150x40/f1f1f1/000000?text=f1f1f1)|12 hex codes where the numbers in the name indicate distance from `divergent_neutral`.|For data has that is bidirectional e.g. Amplification/Deletion values like `seg.mean`|
```

Comment:
I think these should be flipped, as folks tend to expect to see cool colors as lower or negative values and hot colors as higher or positive values. Take GISTIC plots for an example.

##### Amplification q-plot

![image](https://user-images.githubusercontent.com/19534205/76688663-42cfb680-6605-11ea-9bda-c10ca5ef4756.png)

##### Deletion q-plot

![image](https://user-images.githubusercontent.com/19534205/76688672-60048500-6605-11ea-993e-2f8238870ced.png)

Reply:
Okay. Will flip them.